lib/studies/reply/adaptive_nav.dart

```diff
@@ -312,7 +314,12 @@ class _NavigationRailHeader extends StatelessWidget {
             start: 12,
             end: 16,
           ),
-          child: _ReplyFab(extended: extended),
+          child: Consumer<EmailStore>(
+            builder: (context, model, child) {
+              final onMailView = model.currentlySelectedEmailId != -1;
```

Comment:
You have this line in a lot of places, so I suggest you replace them all with with a `model.onMailView` getter in a separate PR

Reply:
Will do thanks!